### PR TITLE
Add ability to disable `whatsThis` button, bump to Swift 5

### PR DESCRIPTION
**Pull request by mbanasiewicz**

### CODE REVIEW

File: Source/UI/Screens/Identifier/IdentifierViewController.swift [hunk 1]
```diff
@@ -21,6 +21,10 @@ class IdentifierViewController: IdentityUIViewController {
 
     @IBOutlet var whastThisButton: UIButton! {
         didSet {
+            guard !configuration.disableWhatsThisButton else {
+                whastThisButton.setTitle("", for: .normal)
```
Comment (mbanasiewicz) [inline Source/UI/Screens/Identifier/IdentifierViewController.swift:25]: This button caused a rejection of our apps b/c the privacy page leads to a page with subscription purchase possibility.

File: SchibstedAccount.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -2280,13 +2280,13 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 11.2;
+				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
```
Comment (mbanasiewicz) [inline SchibstedAccount.xcodeproj/project.pbxproj:2283]: Bumped deployment target to 11.0 as well

File: Source/UI/Screens/Identifier/IdentifierViewController.swift (hunk 1 above)
Comment (zamzterz) [reply Source/UI/Screens/Identifier/IdentifierViewController.swift:25]: Why not hide it with `whastThisButton.isHidden = true` instead?

**Review by zamzterz — COMMENTED** [review]
Many thanks - this is great! 👍 

Some minor things:
* About the hiding of the `whatsThis` button, see review comment.
* Note: I'll merge #309 first so this PR needs to also update `platforms` to `.iOS(.v11)` in `Swift.package` (and fix any eventual merge conflicts :sorry).

File: Source/UI/Screens/Identifier/IdentifierViewController.swift (hunk 1 above)
Comment (mbanasiewicz) [reply Source/UI/Screens/Identifier/IdentifierViewController.swift:25]: Fair point, let me fix that

**Comment by mbanasiewicz** [comment]
@zamzterz Ready 👍

**Pull request merged**